a10_octavia/network/drivers/neutron/a10_octavia_neutron.py

```diff
@@ -42,6 +42,14 @@
 CONF = cfg.CONF
 
 
+class AllocateTrunkException(base.NetworkException):
```

Comment:
You can move `AllocateTrunkException` and `DeallocateTrunkException` to a common `exceptions.py` module now.

Reply:
Let's make that an enhancement task. This branch is pre-rebase so I'd have to go back through and resolve all the merge conflicts.